src/realm/util/serializer.hpp

```diff
@@ -76,6 +77,8 @@ template <>
 std::string print_value<>(realm::UUID);
 template <>
 std::string print_value<>(realm::TypeOfValue);
+template <>
+std::string print_value<>(const realm::Geospatial&);
```

Comment:
Should this go within ` #if REALM_ENABLE_GEOSPATIAL  ... #endif`

Reply:
Yes thanks!